packages/vm/lib/evm/evm.ts

```diff
@@ -385,8 +386,8 @@ export default class EVM {
    * Returns code for precompile at the given address, or undefined
    * if no such precompile exists.
    */
-  getPrecompile(address: Buffer): PrecompileFunc {
-    return getPrecompile(address.toString('hex'), this._vm._common)
+  getPrecompile(address: Address): PrecompileFunc {
+    return getPrecompile(address.buf.toString('hex'), this._vm._common)
```

Comment:
I think it is cleaner if we also make the `precompile` version of `getPrecompile` use `Address` as first argument type instead of a `string`.

Reply:
good idea, missed that on the first pass, done in https://github.com/ethereumjs/ethereumjs-vm/pull/919/commits/17c4dd5255ae8df895fbe29ab5ed466cdd914fd5